docs/components/litellm_chat_model.md

```diff
@@ -0,0 +1,82 @@
+# LiteLLMChatModel
+
+LiteLLM chat model component
+
+## Configuration Parameters
+
+```yaml
+component_name: <user-supplied-name>
+component_module: litellm_chat_model
+component_config:
+  action: <string>
```

Comment:
Shouldn't the load-balancer config show up here?

Reply:
Addressed the comment and unified the configuration. All inferencing and embedding components use the load balancer. Users can set one or more models for the load balancer now.